Common/bkr/common/helpers.py

```diff
@@ -100,7 +101,8 @@ def run(self):
             self.finished.clear()
 
 
-class SensitiveUnicode(unicode):
+@python_2_unicode_compatible
+class SensitiveUnicode():
```

Comment:
Should this be derived from `str` ?

```suggestion
class SensitiveUnicode(str):
```

Reply:
Actually that doesn't change the error ☹️ So never mind.

Reply:
Did not find a way how to make it compatible, so I put it into separate file for now.